docs/overview.md

```diff
@@ -25,14 +25,14 @@ Typically, your component will be made up of the following pieces:
 - Your component code
 - An implementation of the 'BotComponent' class to register your components with the runtime
   - You'll also need to update the `components` array in the `appsettings.json` file with your component name
-- A schema file that declares the inputs and outputs your component requires
-- A `uischema` file to tell Composer where to display your components
+- One or more `.schema` files that declares the inputs and outputs your component requires
```

Comment:
schema files just for BotComponents?  Like it's not needed for declarative packages only?

Reply:
Same as above, but also that why I mention components in this line